src/want-manager/index.js

```diff
@@ -27,12 +45,14 @@ module.exports = class WantManager {
       // add changes to our wantlist
       if (e.cancel) {
         if (force) {
-          this.wantlist.removeForce(e.cid)
+          this.wantlist.removeForce(e.cid.toString())
         } else {
           this.wantlist.remove(e.cid)
         }
       } else {
         this._log('adding to wl')
+        // TODO: Figure out the wantType
+        // @ts-expect-error - requires wantType
         this.wantlist.add(e.cid, e.priority)
```

Comment:
@hugomrdias do you understand why `wantType` is omitted here ? Following lines make make me believe it should be provided, but I'm not sure what the value should be in this instance

https://github.com/ipfs/js-ipfs-bitswap/blob/681b015327a701834b377cce32edbb839cd709b5/src/types/wantlist/index.js#L28-L32

Reply:
I think the WantType ends up defaulting to `Block` if it's omitted.  The `Have`/`Block` distinction is a later optimisation, I think maybe it just got missed off here when it was implemented.